src/utils/regex-constants.js

```diff
@@ -0,0 +1,17 @@
+const reg = require('./reg');
+
+const kurumizawa = /(kurumizawa|胡桃沢)?\s*/;
+const satania = /(satan(i|y|ichi)a|Сатании|サタニキア|サターニャ)\s*-?/;
+const mcdowell = /(mcdowellu?|マクドウェル)?\s*/;
+const honorifics = /(sama|san|chan|senpai|さま|様|さん|ちゃん|せんぱい|先輩)?\s*/;
+const extra = /(!+|！+|¡+|﹗+|︕+|‼+|¿+|？+|⁉+|﹖+|︖+|⁈+|⁇+|\?+|~+|˜+|～+|〜+)?/;
+const sataniaName = reg`${kurumizawa}${satania}${mcdowell}${honorifics}`;
+
+module.exports = {
```

Comment:
you could use the shorthand notation, like `{ kurumizawa, satania, mcdowell, honorifics, extra, sataniaName }`